src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sat calendar--start-day-30">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sat calendar--start-day--30">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/9b354ae6-b2a9-41ed-8113-a267f726695a)